### DISCUSSION TITLE
CTSM5.3 historical simulations: Known issues

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/LMWG_dev#71] [General]
These thoughts come from from looking at [diagnostics](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/ctsm53n04ctsm52028_f09_hist_2004_2023-ctsm52026_f09_hist_2004_2023/setsIndex.html) and [ILAMB results](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/_build_ctsm53n04ctsm52028_f09_hist/) posted with #69.

## Issues we may want to address with the PPE or hand calibration
- [ ] **Dead grasses** (see maps below, but [also posted here](https://github.com/NCAR/LMWG_dev/issues/69#issuecomment-2386331653), related to FUN bug  or fire parameterization of `froot_leaf`?)
- [ ] **High LAI** in: 
  - tropical deciduous forests (related to fire parameterization of `froot_leaf`?)
  - larches (deciduous needleleaf trees)
- [ ] [**Permafrost extent**](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/_build_ctsm53n04ctsm52028_f09_hist/HydrologyCycle/Permafrost/Obu2018/Obu2018.html#AllModels) (maybe we're OK, or consider modifying Sturm thermal conductivity parameterization)?
- [ ] [**Water stress** in the Amazon](https://github.com/NCAR/LMWG_dev/issues/69#issuecomment-2378165543) (related to ad-hoc kmax parameter change to increase ET and reduce LAI)
- [ ] [**High vegetation biomass**](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/_build_ctsm53n04ctsm52028_f09_hist/EcosystemandCarbonCycle/Biomass/GlobalCarbon/GlobalCarbon.html), especially in the tropics (increase rmort, modify LAI and GPP?) ?
- [ ] [**Fire evaluation**] including [fire C loss](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/ctsm53n04ctsm52028_f09_hist_2004_2023-ctsm52026_f09_hist_2004_2023/set1/set1_COL_FIRE_CLOSS.gif) & [fraction burned area](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/ctsm53n04ctsm52028_f09_hist_2004_2023-ctsm52026_f09_hist_2004_2023/set2/set2_ANN_FAREA_BURNED.gif)
- [ ] Crop model calibration (not listed here).

![image](https://github.com/user-attachments/assets/28acd2f7-57ba-4a62-825c-5bedbb5562c0)

-

**Comment by samsrabin** [comment]
Could you point me toward the diagnostic plots of a run with "better" C4 Grass distribution?

**Reply by wwieder** [reply]
do you mean a run with living C4 grass?

**Reply by samsrabin** [reply]
Yeah, exactly.

**Reply by wwieder** [reply]
These are [GSWP3 forced runs with CLM5.1](https://www2.cgd.ucar.edu/tss/clm/diagnostics/clm5.0_ctsm5.1/pft_level_plots/ctsm51_cesm23a02cPPEn08ctsm51d030_1deg_GSWP3V1_hist_pft_2dPDF_whittaker-1.png) (therefore dated), where C4 grasses still don't look great, but less bad than they are now.  

@olyson will look at the TRENDY runs that Adrianna did with 5.3 datasets, CRUJRA datm & the older parameter file (w/ out froot_leaf changes from fire work.

**Reply by djk2120** [reply]
I looked at some PPE runs we had (GSWP3, CLM5.1 default params) with the FUNbug and then with that rectified. Fixing the FUNbug reduced C4 LAI by around 33% on average, but most of your problem gridcells were already dead. 

<img src="https://github.com/user-attachments/assets/7a062490-9964-41c0-bca8-ef6059e0cc28" alt="scatter" width="400"/>

![funbug_deadpixels](https://github.com/user-attachments/assets/60d21e8a-5611-4c33-96d6-e950bd93552a)

**Comment by olyson** [comment]
Survivability and Max TLAI for the TRENDY run (TRENDY2024_f09_clm60_S3):
![TRENDY2024_f09_clm60_S3_pft_2dPDF_whittaker-0](https://github.com/user-attachments/assets/5b657d8c-18b0-48bd-ad97-44f54d8c23c9)
![TRENDY2024_f09_clm60_S3_pft_2dPDF_whittaker-1](https://github.com/user-attachments/assets/7a6db696-5300-4734-99cd-704b07164161)

**Reply by wwieder** [reply]
Thanks Keith.  C4s look pretty dead before changes to `froot_leaf`.  Hopefully this is something the PPE tools can help address?

**Comment by olyson** [comment]
I was looking for GSWP3 driven historicals that are newer than the CLM5.1 simulation that @wwieder linked above and saw that I had a historical that was used for nuopc validation.  It used cesm2_3_alpha03a (ctsm5.1.dev038).  Below are survivability and max LAI plots.  Survivability looks better than the TRENDY simulation (84% versus 62% in the TRENDY simulation) and LAI is higher.
There obviously are science differences between these tags, but there are parameter differences for C4 grasses as well (including froot_leaf as we know), summarized here:
```
            NEW    OLD
cc_leaf     0.6    0.8
cc_other    0.5    0.8
fm_leaf     0.2    0.8
fm_other    0.2    0.8
froot_leaf  0.5    1.5
fsr_pft     0.31   0.32
kc_nonmyc   0.12   7.2
kn_nonmyc   7.2    0.12
```
![ctsm51_cesm23a03a_1deg_GSWP3V1_nuopc_hist_pft_2dPDF_whittaker-0](https://github.com/user-attachments/assets/929d798e-5e2d-4a17-ab0a-2ec03309f091)

![ctsm51_cesm23a03a_1deg_GSWP3V1_nuopc_hist_pft_2dPDF_whittaker-1](https://github.com/user-attachments/assets/b47eada4-a44f-4aa4-9081-38cbd828ad78)

**Reply by wwieder** [reply]
Thanks for digging in here Keith. My other suspicion was that the FUN bug that increased n costs killed the C4 grasses. In CLM5 they were strongly n limited, correcting the fun bug may have done them in?

**Comment by samsrabin** [comment]
@lifang0209 noticed a large excess of burned area on the Tibetan Plateau. From a comment on [Issue #2728: Incorporate the impact of O2 concentration on fuel flammability](https://github.com/ESCOMP/CTSM/issues/2728), which proposes a parameterization to handle this:

> The attached figure displays the spatial distribution of the annual burned area fraction. Observations (obs) represent the average of three satellite-based burned area products (GFED5s, FireCCI, and MODIS C6). Simulations show results for the new fire schemes both with and without an O₂ limit. The figure includes global burned area totals and spatial correlation coefficients (R) between observations and simulations. As illustrated, the O₂ limit reduces the overestimation of the burned area fraction in Tibet, and improves the spatial pattern simulation performance (global spatial correlation with observations increases from 0.76 to 0.79). ![o2limit](https://private-user-images.githubusercontent.com/30509313/371160368-7154193d-fab9-40dc-94ed-6b89a94ef2d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0MjkxNDUsIm5iZiI6MTczMTQyODg0NSwicGF0aCI6Ii8zMDUwOTMxMy8zNzExNjAzNjgtNzE1NDE5M2QtZmFiOS00MGRjLTk0ZWQtNmI4OWE5NGVmMmQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEyVDE2MjcyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2MDVmZmVkMTQ5NjY2YWFjNDg0YTE2MmIyY2JmY2UxOTM2MWIwZjJhY2U4NGM5ZmYwYTRjOTcwMTJiNjUwOTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VV4GQ8yGOBcLh65lYb-fK7d5Zoui7nD024RrXRv7oMc)

**Comment by wwieder** [comment]
Quick update on survivability from #74 that includes updates to:
- Nfix temperature function and 
- Arctic FUNfrac_fixers
- Backs off arctic tuning from the 5.3.0
- Reverts froot_leaf in C4 grass and tropical deciduous trees (from Fang's tuning)
- Makes N update cheeper for C4 grasses

My assessment, compared to results in #70 (first firgure below), results from #74 (second figure below)
- Arctic grass and shrubs looks good
- Tropics look much better w/ C4 living and deciduous trees less absurd
- C3 grasses look really different that C3 generic crops.  Why is this?
- Changes to boreal evergreen trees likely need to be backed off too, with lower N fix in the arctic.

**Reply by lifang0209** [reply]
Does "Reverts froot_leaf in C4 grass and tropical deciduous trees (from Fang's tuning)" mean you prefer to set froot_leaf back to 1.5? Setting froot_leaf=0.5 is beneficial for their growth and post-fire recovery.

**Reply by wwieder** [reply]
This may be true, but the parameter change seems to contribute to high LAI biases in deciduous trees (mean max LAI > 12!)  and may contribute to C4 grass mortality (especially in Africa).

![image](https://github.com/user-attachments/assets/051e9224-f1b8-4675-9e0c-19e18f08ebd5)

**Reply by wwieder** [reply]
Modifying `froot_leaf` and reducing C4 grass N uptake costs seems to help with these biases.  

![image](https://github.com/user-attachments/assets/04aec6dd-1181-467f-8e4f-218ef85462c8)
